### ISSUE TITLE
Unable to build on Win10

### ISSUE THREAD

**Issue by Slappy12** [tobspr/shapez.io#1284]
Everytime I run yarn gulp it says not a git repository an stops. ffmpeg is installed.

[LOg.txt](https://github.com/tobspr/shapez.io/files/6983297/LOg.txt)

**Comment by EmeraldBlock** [comment]
did you `git clone` the repo?

**Comment by Slappy12** [comment]
No, i did not clone the repo. But in Gitpod it works perfectly fine but on my computer it does not work.

**Comment by EmeraldBlock** [comment]
er, that's the issue lol (Gitpod does it "all for you")

**Comment by slooi** [comment]
As EmeraldBlock said, make sure you `git clone`. Do not download as zip. Downloading the repo as a zip will not work
